src/lib.rs

```diff
@@ -204,6 +204,7 @@ pub use element::{
     reader::ElementReader, writer::ElementWriter, Annotations, Element, IntoAnnotatedElement,
     IntoAnnotations, Sequence, Value,
 };
+pub use ion_data::ion_eq::IonEq;
```

Comment:
Let's not expose this publicly (at least not yet). There was a lot of back and forth about this when it was implemented, and I can't remember all of the details right now. I don't think we should export this without revisiting the prior discussion.